cmd/create/quick-start-cluster/vpc.yaml

```diff
@@ -0,0 +1,111 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CloudFormation template equivalent to the provided Terraform code
+
+Parameters:
+  NamePrefix:
+    Type: String
+    Description: Prefix for naming resources
+  VpcCidr:
+    Type: String
+    Description: CIDR block for the VPC
+  AvailabilityZonesCount:
+    Type: Number
+    Description: Number of availability zones
+  Tags:
+    Type: CommaDelimitedList
+    Description: Additional tags
+
+Conditions:
+  HasTags: !Not [!Equals [!Join [",", !Ref Tags], ""]]
+
+Resources:
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: !Ref VpcCidr
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      Tags:
+        - Key: Name
+          Value: !Sub "${NamePrefix}-vpc"
+        - Key: AdditionalTags
+          Value: !If 
+            - HasTags
+            - !Join [",", !Ref Tags]
+            - ""
+
+  S3VPCEndpoint:
+    Type: AWS::EC2::VPCEndpoint
+    Properties:
+      VpcId: !Ref VPC
+      ServiceName: !Sub "com.amazonaws.${AWS::Region}.s3"
+
+  PublicSubnets:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: "https://quickstart-creation-bucket.s3.us-west-2.amazonaws.com/templates/public_subnets.yaml"
```

Comment:
Is there any reason for making these templates instead of a single file? 

I am just wondering if there is an easy way to bubble up the outputs into a single stack?